go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	github.com/zclconf/go-cty v1.10.0
 	go.opentelemetry.io/otel v1.4.1
 	go.opentelemetry.io/otel/trace v1.4.1
+	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd // indirect
```

Comment:
temporarily adding this as "explicit"; this will dissolve once buildkit is updated with https://github.com/moby/buildkit/pull/2738